bBridgeAPISDK.Test/TestAuthorization.cs

```diff
@@ -11,15 +12,18 @@ public class TestAuthorization
         [Fact]
         public void TestCanReceiveAuthorizationTokenForUserNameAndPassword()
         {
-            var baseUri = new Uri(Resources.bBridgeAPIBaseURI);
-
             IAuthorizer userPasswordAuthorizer = new LazyCredentialsAuthorizer(
-                Resources.bBridgeAPIUserName,
-                Resources.bBridgeAPIPassword,
-                baseUri);
+				string.IsNullOrEmpty(TestResources.bBridgeAPIUserName) ?
+					Environment.GetEnvironmentVariable("BBRIDGE_API_USER_NAME") : 
+					TestResources.bBridgeAPIUserName,
+				string.IsNullOrEmpty(TestResources.bBridgeAPIUserName) ?
+					Environment.GetEnvironmentVariable("BBRIDGE_API_PASSWORD") :
+					TestResources.bBridgeAPIPassword,
+                Path.Combine(TestResources.bBridgeAPIBaseURI,
+                TestResources.bBridgeAPIAuthUrlSuffix));
 
-            Assert.True(Regex.IsMatch(userPasswordAuthorizer.Token, Resources.JWTTokenRegex));
             Assert.Equal(userPasswordAuthorizer.Token.Length, 186);
```

Comment:
Why do you consider JWT token length is 186? It isn't true because length depends on payload that can have an unlimited number of fields